deploy/cfn.yml

```diff
@@ -28,6 +28,9 @@ Parameters:
   EnvironmentName:
     Type: String
     Description: "The name of the environment.  Use \"prod\" without quotes for the production environment."
+  ErrorAlarmTopicArn:
+    Type: String
+    Default: arn:aws:sns:eu-west-2:390403885523:error-alarm
```

Comment:
Can this also be defined by the template?

Reply:
Would it require us to confirm after each deployment, or only when a change is made to the topic in the template?